ee/modules/600-flant-integration/images/flant-pricing/hooks/controller_metrics.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+#
+# This is a config for controller_metrics.py hook.
+
+configVersion: v1
+kubernetes:
+  - name: deploy
+    queue: /controller_metrics_deploy
+    apiVersion: apps/v1
+    kind: Deployment
+    group: main
+    executeHookOnEvent: []
+    executeHookOnSynchronization: false
+    keepFullObjectsInMemory: false
+    jqFilter: |
+      {
+        "controller_name": .metadata.name,
```

Comment:
Nitpicking. Do we need to repeat `controller_` prefix? I think we know the context well enough to be shorter.